rtgui/controlspotpanel.cc

```diff
@@ -402,12 +404,29 @@ ControlSpotPanel::ControlSpotPanel():
     avoidmunConn_  = avoidmun_->signal_toggled().connect(
                       sigc::mem_fun(*this, &ControlSpotPanel::avoidmunChanged));
     
-    Gtk::Frame* const avFrame = Gtk::manage(new Gtk::Frame());
+    Gtk::Label* const labelgamut = Gtk::manage(new Gtk::Label(M("TP_LOCALLAB_GAMUTTYPE") + ":"));
+    ctboxgamut->pack_start(*labelgamut, Gtk::PACK_SHRINK, 4);
+    avoidgamutMethod_->append(M("TP_LOCALLAB_GAMUTNON"));
+    avoidgamutMethod_->append(M("TP_LOCALLAB_GAMUTLABRELA"));
+    avoidgamutMethod_->append(M("TP_LOCALLAB_GAMUTXYZABSO"));
+    avoidgamutMethod_->append(M("TP_LOCALLAB_GAMUTXYZRELA"));
+    avoidgamutMethod_->append(M("TP_LOCALLAB_GAMUTMUNSELL"));
+    avoidgamutMethod_->set_active(4);
+    avoidgamutconn_ = avoidgamutMethod_->signal_changed().connect(
+                     sigc::mem_fun(
+                         *this, &ControlSpotPanel::avoidgamutMethodChanged));
+    ctboxgamut->pack_start(*avoidgamutMethod_);
+    if (showtooltip) {
+        ctboxgamut->set_tooltip_text(M("TP_LOCALLAB_GAMUT_TOOLTIP"));
+    }
+
+ Gtk::Frame* const avFrame = Gtk::manage(new Gtk::Frame());
     ToolParamBlock* const avbox = Gtk::manage(new ToolParamBlock());
     avFrame->set_label_align(0.025, 0.5);
-    avFrame->set_label_widget(*avoid_);
+   // avFrame->set_label_widget(*avoid_);
+    avbox->pack_start(*ctboxgamut);
     avbox->pack_start(*avoidrad_);
-    avbox->pack_start(*avoidmun_);
+    //avbox->pack_start(*avoidmun_);
```

Comment:
Better to remove `avoid_` and `avoidmun_` completely to avoid creating unnecessary widgets and causing memory leaks. Even with `Gtk::manage`, GTK will never delete the widgets because they are not attached to anything.
https://github.com/Beep6581/RawTherapee/blob/6f2ca3f2581e6f6dcecb00ffb6bb6a730988c972/rtgui/controlspotpanel.cc#L87-L88

Reply:
done... I hope :)